src/mpid/ch4/src/ch4_vci.h

```diff
@@ -60,10 +60,10 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_get_vci(int flag, MPIR_Comm * comm_ptr,
                                            int src_rank, int dst_rank, int tag)
 {
     if (flag & 0x1) {
-        /* src */
+        /* dst */
         return (tag == MPI_ANY_TAG) ? 0 : ((tag >> 10) & 0x1f);
```

Comment:
Swap the `condition` in stead. The intention was to have the first 5 bits for `src` and the next 5 bits for `dst`, and the last 5 bits for user.

Reply:
I see. Done.